<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ONNXExport</class>
 <widget class="QDialog" name="ONNXExport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ONNX Export - Visionin</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>351</width>
     <height>56</height>
    </rect>
   </property>
   <property name="title">
    <string>Input Model</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton_yolov4">
      <property name="text">
       <string>YoloV4-DarkNet</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_yolov7">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>YoloV7-Pytorch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_efficientdet">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>EfficientDet</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Model Path</string>
   </property>
   <widget class="QComboBox" name="comboBox_path">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>27</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dir">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>26</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Path contain cfg, weights, names file</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_xport">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
